litmus-portal/backend/subscriber/igc/graphql/subscriptions/clusterSubscription.js

```diff
@@ -0,0 +1,30 @@
+const appolo = require('apollo-link');
+const gql = require('graphql-tag');
+
+const operation = () => ({
+    query: gql`
+		subscription {
+			clusterSubscription {
+				id
+				data
+			}
+		}
+	`,
+});
+
+const nextHandler = (ws, response) => {
+    console.log(JSON.parse(response.data.clusterSubscription.data));
+    ws.send(JSON.stringify(response.data.clusterSubscription));
+};
+const errHandler = (ws, err) => {
+    console.log(err);
+    ws.send(JSON.stringify({ type: 'error', data: 'Error' }));
+};
+
+module.exports = (link, ws, {} = {}) => {
```

Comment:
We can remove the data field if we don't need it, or just use data={} instead of {}={}